src/DefinesRoutes.php

```diff
@@ -17,9 +17,11 @@ public function ensureRoutesAreDefined()
             return;
         }
 
-        Route::post(
-            '/vapor/signed-storage-url',
-            Contracts\SignedStorageUrlController::class.'@store'
-        )->middleware(config('vapor.middleware', 'web'));
+        if (config('vapor.define_routes', true)) {
```

Comment:
There is other routes on this package, so maybe make this key more specific?

Reply:
Hmm my bad, I dunno why I thought this was the only place that defines routes... :) I'll update this for sure. 👍

Reply:
@nunomaduro I updated the toggle control so it's called `vapor.signed_storage.enabled`.

I also added config control for the actual URL, which is defaulting to the current value, and it's called `vapor.signed_storage.url`.

This way you can easily turn it on or off, or change the actual route where you want this to point. Is this ok or should I remove the URL config control?